Cargo.toml

```diff
@@ -159,6 +159,7 @@ quote = "1.0"
 rand = "0.8"
 rand_chacha = "0.3"
 reedline = "0.22"
+regex = "1.9"
```

Comment:
Is this still used?

Reply:
I removed it and it broke. Now that I've rebased I'll try it again

Reply:
Good pickup. It snuck back in because a few rebases failed. I'm surprised Clippy didn't complain. Fixed.

Reply:
I have a huge problem with regex in security applications. It's the source of all evil.